client/jetpack-connect/user-type/form.jsx

```diff
@@ -0,0 +1,89 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import { recordTracksEvent } from 'state/analytics/actions';
+
+/**
+ * Style dependencies
+ */
+import '../style.scss';
+
+class UserTypeForm extends Component {
+	static propTypes = {
+		submitForm: PropTypes.func.isRequired,
+
+		// from localize() HoC
+		translate: PropTypes.func.isRequired,
+	};
+
+	constructor( props ) {
+		super( props );
+
+		this.state = {
+			userType: '',
+		};
+	}
+
+	handleRadioChange = event => this.setState( { userType: event.currentTarget.value } );
+
+	handleSubmit = userType => {
+		//	const { userType } = this.state;
```

Comment:
Is this still WIP? Let's fix it, or If we have plans to update it in a future PR, let's add a TODO comment.

Reply:
missed to remove it.oops